builtin/providers/postgresql/resource_postgresql_db_funcs.go

```diff
@@ -0,0 +1,76 @@
+package postgresql
+
+import (
+	"fmt"
+	"database/sql"
+
+	"github.com/lib/pq"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourcePostgresqlDbCreate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
Well I did see some providers which had these split up and it seemed to me a bit easier to read that way, not sure if there's a better rationale for the other approach.